<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="whatsThis">
   <string>&lt;i&gt;Move the mouse cursor over a settings item to get more information.&lt;/i&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="highlightValidTargetsCheckBox">
         <property name="whatsThis">
          <string>When checked, the square under the cursor is highlighted if it's a valid target for the move.</string>
         </property>
         <property name="text">
          <string>Highlight valid target &amp;squares under the cursor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="showNotificationsCheckBox">
         <property name="whatsThis">
          <string>When checked, a notification will be shown on an invalid move or if the game ends.</string>
         </property>
         <property name="text">
          <string>Show &amp;notifications</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="resumeGameAtStartupCheckBox">
         <property name="whatsThis">
          <string>When checked, saves the game when exiting, and resumes it the next time. If other sessions of the program are open, only one of them will do this; the others may ask for confirmation when exiting.</string>
         </property>
         <property name="text">
          <string>&amp;Resume game at program startup</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="languageTab">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="interfaceLanguageLabel">
         <property name="text">
          <string>Interface &amp;language:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="languageListContainer"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="computerTab">
      <attribute name="title">
       <string>Computer</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="difficultyLabel">
           <property name="text">
            <string>&amp;Difficulty level:</string>
           </property>
           <property name="buddy">
            <cstring>searchDepthSlider</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="searchDepthSlider">
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="difficultyValueLabel">
           <property name="text">
            <string notr="true">&lt;?&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphicsTab">
      <attribute name="title">
       <string>Graphics</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="hardwareAccelerationCheckBox">
         <property name="whatsThis">
          <string>Use OpenGL for hardware acceleration when available.</string>
         </property>
         <property name="text">
          <string>Use &amp;hardware acceleration when available</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="disableAnimationsCheckBox">
         <property name="text">
          <string>Disable &amp;animations</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="updateTab">
      <attribute name="title">
       <string>Update</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkForUpdatesCheckBox">
         <property name="whatsThis">
          <string>When checked, you will be notified when a new version of the program is available.</string>
         </property>
         <property name="text">
          <string>&amp;Check for updates</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="oncePerLabel">
           <property name="text">
            <string>&amp;Once per</string>
           </property>
           <property name="buddy">
            <cstring>updateCheckIntervalSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="updateCheckIntervalSpinBox"/>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>25</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="daysLabel">
           <property name="text">
            <string notr="true">days</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="whatsThisLabel">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>7</number>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>highlightValidTargetsCheckBox</tabstop>
  <tabstop>showNotificationsCheckBox</tabstop>
  <tabstop>searchDepthSlider</tabstop>
  <tabstop>hardwareAccelerationCheckBox</tabstop>
  <tabstop>disableAnimationsCheckBox</tabstop>
  <tabstop>checkForUpdatesCheckBox</tabstop>
  <tabstop>updateCheckIntervalSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
